docs/how_to/ssh_connection_preference.md

```diff
@@ -30,7 +30,10 @@ HOSTS:
     hypervisor: vmpooler
     platform: ubuntu-16.04-amd64
     template: ubuntu-1604-x86_64
-    ssh_preference: [:vmhostname, :hostname, :ip]
```

Comment:
Alternately you could do:
```suggestion
    ssh_preference: '[:vmhostname, :hostname, :ip]'
```
But yeah this is some strange yaml parsing behavior. https://github.com/ruby/psych/issues/228

Reply:
hm, actually no that suggestion does not work (the value is just a string). So yeah what you have here is probably best.